lib/gui/pages/finish/styles/_finish.scss

```diff
@@ -26,7 +26,9 @@
 }
 
 .page-finish .button-primary {
-  min-width: $btn-min-width;
```

Comment:
I'm also happy with a new variable called `$btn-large-min-width` or something like that. Basically my point is to try to not make many different exceptions to something like a button width on different parts of the app.